src/applications/vaos/utils/eligibility.js

```diff
@@ -150,7 +150,7 @@ function hasDirectFailed(eligibilityData) {
  * one block of checks is successful, we can still let a user continue on that,
  * even if another path is blocked.
 */
-export function getEligibilityChecks(systemId, typeOfCareId, eligibilityData) {
```

Comment:
These params weren't being used, but ESLint didn't care because they last one is